src/scripts/components/main/app/app.tsx

```diff
@@ -60,7 +60,7 @@ const TranslatedApp: FunctionComponent = () => {
             <div className={styles.logo}>
               <EsaLogo />
             </div>
-            <Globes markers={markers} />
+            <Globes markers={markers} backgroundColorString={'#10161A'} />
```

Comment:
sorry for another comment 🙈  but could you remove the "String" from "backgroundColorString". the typescript type already defines that it is a string

Reply:
Haha no problem. I named it _backgroundColorString_ because i'm using _backgroundColor_ already in globe.tsx:
`const backgroundColor = Color.fromCssColorString(backgroundColorString);`

How about something like backgroundColorCode or backgroundColorHex, then?

Reply:
I would prefer to rename the inner variable instead. The property names are more important to be precise because it's the interface for someone who uses the component and doesn't (need to) know whats going on inside. 
Also in this case the inner variable is not needed to be defined as it is only used once. Just set the color directly:
```js
scopedViewer.scene.backgroundColor = Color.fromCssColorString(backgroundColorString);
```

Reply:
True, good point. :) I'll change it, thanks.